pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -350,12 +293,21 @@ def list_remote_files(tag, sat_id,
     search_dir = futils.construct_searchstring_from_format(format_dir)
     n_layers = len(search_dir['keys'])
 
-    # only keep file portion of format
+    # Only keep file portion of format
     format_str = dir_split[-1]
     # Generate list of targets to identify files
     search_dict = futils.construct_searchstring_from_format(format_str)
     targets = [x.strip('?') for x in search_dict['string_blocks'] if len(x) > 0]
 
+    # Remove any additional '?' characters that the user may have supplied
+    new_targets = []
+    for target in targets:
+        tstrs = target.split('?')
+        for tstr in tstrs:
+            if tstr != '':
+                new_targets.append(tstr)
```

Comment:
```suggestion
If all `?` are removed then this could be done via:
```suggestion
        new_targets.append(target.replace('?', ''))
``````

Reply:
Not quite the same.  If `targets = ['de2_neutral?s_nacs_', '_v', '.cdf']`, we want new_targets to be `['de2_neutral', 's_nacs_', '_v', '.cdf']` not `['de2_neutrals_nacs_', '_v', '.cdf']`.  This whole routine could be simplified though, and could use a scrubbing in a future version.